worker/src/handlers/schedule.ts

```diff
@@ -73,15 +73,15 @@ async function processQueue(): Promise<void> {
 
     core_analytics[timestampString] = queue_data;
 
+    // Trigger Netlify build
+    await fetch(NETLIFY_BUILD_HOOK, { method: "POST" });
+
     await KV.put(
       `${KV_PREFIX_HISTORY}:${timestampString}`,
       JSON.stringify(queue_data)
     );
     await KV.put(KV_KEY_CORE_ANALYTICS, JSON.stringify(core_analytics));
```

Comment:
I would do this write still before the trigger. And I would also check the return value of the netlify trigger and raise if `resp.ok` is false.

Reply:
We have to do everything or nothing.